exchanges/coinbasepro/coinbasepro_websocket.go

```
@@ -515,3 +503,30 @@ func base64URLEncode(b []byte) string {
 	s = strings.ReplaceAll(s, "/", "_")
 	return s
 }
+
+// checkSubscriptions looks for incompatible subscriptions and if found replaces all with defaults
```

I don't really understand the point of this:

1. If they're incompatible with the exchange, since they're not the names of actual channels, why not correct them, or let the exchange throw an error, instead of silently replacing the call with defaults?
2. If they're meant to be incompatible with each other, why does it only check for the presence of one, and not the presence of multiple simultaneously?

On another note, this removes the nil pointer check the general checkSubscriptions() function has, so it will panic on c.Config or c.Config.Features being nil. Is that worth addressing?

---

1) Because the websocket isn't actually working right now, we can use any of the incompatible ones to spot that the  _entire_ config is completely out-of-date, and then upgrade-and-replace the whole thing, including adding new ones.

Equivalent to saying "You have config v1. That's now entirely unusable. Replacing it with v2".

Because Coinbase WS isn't working, it doesn't seem likely anyone has tried customising these subscriptions, so we're okay to just replace them wholesale.